src/migration/updateProfile.js

```diff
@@ -0,0 +1,113 @@
+const _ = require("lodash");
```

Comment:
@pallakartheekreddy , migration script need not be committed no?  we will run it locally?